go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
 	github.com/armon/go-radix v1.0.0 // indirect
 	github.com/certifi/gocertifi v0.0.0-20180905225744-ee1a9a0726d2 // indirect
-	github.com/containers/buildah v1.11.7-0.20191206171145-8d6869bccee0
-	github.com/containers/image/v5 v5.0.0
-	github.com/containers/storage v1.15.3
+	github.com/containers/buildah v1.11.6-0.20200109175634-4e23b7a10cc9
```

Comment:
Per discussion in https://github.com/openshift/builder/pull/127/files#r366946525, this is ok.